ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/Metric.scala

```diff
@@ -140,22 +176,36 @@ object Metric {
       )
     }
 
-    override def periodicUpdate(): (ConsumptionSpeedMetric[T], String) = {
-      val update = s"speed (interval): ${periodicSpeed.map(rounded).getOrElse("-")} [-]"
+    override def periodicValue(): (Metric[T], MetricValue) = {
+      val value: Option[Double] = periodicSpeed
```

Comment:
Does it bring anything if you use this intermediate value?

Reply:
No, it doesn't. Good catch!